src/tls_wrap.cc

```diff
@@ -881,6 +889,90 @@ int TLSWrap::SelectSNIContextCallback(SSL* s, int* ad, void* arg) {
 #endif  // SSL_CTRL_SET_TLSEXT_SERVERNAME_CB
 
 
+#ifdef OPENSSL_PSK_SUPPORT
+
+unsigned int TLSWrap::PskServerCallback(SSL *ssl,
```

Comment:
C++ doesn't call `pskCallback` directly, but `onpskexchange`. So the `SecureContext` instance doesn't leak into the API.

But I think `pskCallback` context right now is the options object it sits on. However, for `tls.createServer`, this is no the user options object, but an server internal one.